src/typings/prop-types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'prop-types' {
+    import React = require('react');
+
+    export const bool: {
+        isRequired: React.Validator<any>
```

Comment:
Plus: I ran into a TS compiler error when trying to tsc "error TS2312: An interface may only extend a class or another interface." so I had to modify the code a little.